lib/src/importer-registry.ts

```diff
@@ -11,6 +11,28 @@ import {FileImporter, Importer, Options} from './vendor/sass';
 import * as proto from './vendor/embedded_sass_pb';
 import {catchOr, thenOr, PromiseOr} from './utils';
 
+const entryPointDirectoryKey = Symbol();
+
+export class NodePackageImporter {
+  [entryPointDirectoryKey]?: string;
```

Comment:
I believe this can be `readonly` and non-nullable, since it's always set in the constructor (if it doesn't throw an error).

Reply:
I'd definitely prefer to have this readonly and non-nullable, since that's a direct representation of the actual state of the class. The static errors raise an interesting issue, though—because TypeScript checks types structurally, and the existing `NodePackageImporter` type from the Sass package has no properties, it will effectively allow _any_ object to be passed as an importer. It should probably have its own secret symbol property to avoid this, which will in turn mean that we will need a cast here no matter what.

Reply:
@nex3 Ah, ok -- that makes sense. So adding a private readonly non-nullable Symbol key to the definition in the Sass repo, correct?